opm/autodiff/BlackoilWellModel_impl.hpp

```diff
@@ -757,11 +795,16 @@ namespace Opm {
     BlackoilWellModel<TypeTag>::
     recoverWellSolutionAndUpdateWellState(const BVector& x)
     {
-        if (!localWellsActive())
-            return;
+        Opm::DeferredLogger local_deferredLogger;
 
-        for (auto& well : well_container_) {
-            well->recoverWellSolutionAndUpdateWellState(x, well_state_);
+        if (localWellsActive()) {
+            for (auto& well : well_container_) {
+                well->recoverWellSolutionAndUpdateWellState(x, well_state_, local_deferredLogger);
```

Comment:
Can throw, need to catch and communicate.

Reply:
done